### PR TITLE
Detailed description

### PR DESCRIPTION
**Pull request by maje02**
I added details about each Place to the tiles. The contact information is interactive (a click on the phone number calls the place, a click on Mail opens the email-program, a click on Website opens the Website in a new page).

### CODE REVIEW

File: src/City.js [hunk 1]
```diff
@@ -14,17 +14,39 @@ export default function City({ places }) {
...
+            mail,
+            websiteURL,
+            opening_times,
+            description,
+            categories,
+          },
+          index
+        ) => (
+          <Place
+            key={index}
```
Comment (joshualang) [inline src/City.js]: (May get problems with index key later on. Better to use some sort of id.)

File: src/Place.js [hunk 2]
```diff
@@ -19,14 +24,26 @@ export default function Place({
...
       </PlaceAddress>
+      <ContactDetails>
+        <Contact href={'tel:' + phone}>{phone}</Contact>
+        <Contact href={'mailto:' + mail}>Mail</Contact>
+        <Contact href={websiteURL} target="_blank">
+          Website
+        </Contact>
+      </ContactDetails>
+      <Times>
+        Öffnungszeiten: <br></br>
```
Comment (joshualang) [inline src/Place.js]: two <br> tags = not best practice. consider using an extra <p> tag instead.

File: src/Place.js [hunk 3]
```diff
@@ -19,14 +24,26 @@ export default function Place({
...
     </PlaceBody>
   )
 
   function renderCategories(categories, index) {
-    return categories.map(category => (
-      <PlaceCategory key={index}>{category}</PlaceCategory>
-    ))
+    return categories
+      .sort((a, b) => a.localeCompare(b))
+      .map(category => <PlaceCategory key={index}>{category}</PlaceCategory>)
```
Comment (joshualang) [inline src/Place.js]: Movie the Place function into a new file

File: src/City.js (hunk 1 above)
Comment (maje02) [reply src/City.js]: will do this later in the project

**Pull request merged**